steps/tslint/0.0.2/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Run tslint command
+summary: Runs `tslint` with the given args.
+description: Runs `tslint` with the given args.
+website: https://github.com/ibratoev/steps-tslint
+source_code_url: https://github.com/ibratoev/steps-tslint
+support_url: https://github.com/ibratoev/steps-tslint/issues
+published_at: 2017-03-22T16:03:54.333538005+02:00
+source:
+  git: https://github.com/ibratoev/steps-tslint.git
+  commit: 764d22569364b38e572a1f0c8783e26beca79aee
+host_os_tags:
+- osx
+- ubuntu
+project_type_tags:
```

Comment:
following project types are available on bitrise:

- ios
- android
- xamarin
- react-native
- cordova
- ionic

is this step available only for ios and android?